test/Microsoft.ML.Tests/Transformers/DateTimeTransformerTests.cs

```diff
@@ -23,7 +23,7 @@ private class DateTimeInput
             public long date;
         }
 
-        [NotCentOS7Fact]
```

Comment:
(minor) Can the NotCentOS7Fact attribute be removed now? I guess it was only used for "Featurizers" before...?

Reply:
Correct. It is now removed.